core/src/main/assets/favorites-export-template.html

```diff
@@ -0,0 +1,82 @@
+<?xml version='1.0' encoding='UTF-8' standalone='no' ?>
+<html>
+    <head>
+        <title>AntennaPod Favorites</title>

```

Comment:
This file is more or less a duplicate of the feed export feature. Could you please use a single file only (maybe with placeholder for the title)? That reduces problems when modifying just one of the files.

Reply:
Legacy code habits... "do not touch the arcane code". But your suggestion is logical :-)